pydantic_settings_export/utils.py

```
@@ -206,35 +211,186 @@ def _find_settings_in_module(module: ModuleType) -> list[type[BaseSettings]]:
+
+
+def _import_module_from_file(path: Path, project_dir: Path | None = None) -> ModuleType:
+    """Import a module from a Python file path."""
+    importlib.invalidate_caches()
+
+    module_name = _make_module_name(path, project_dir)
+    if module_name:
+        with _prepend_sys_path(project_dir):
+            try:
+                return importlib.import_module(module_name)
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)